test/e2e/testdata/httproute-routingtype.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: gateway.envoyproxy.io/v1alpha1
+kind: EnvoyProxy
+metadata:
+  name: custom-proxy-config
+  namespace: gateway-conformance-infra
+spec:
+  routingType: service
+---
+apiVersion: gateway.networking.k8s.io/v1
+kind: HTTPRoute
+metadata:
+  name: httproute-to-backend-fqdn
+  namespace: gateway-conformance-infra
+spec:
+  parentRefs:
+  - name: same-namespace
+  rules:
+  - matches:
+    - path:
+        type: PathPrefix
+        value: /backend-fqdn
+    backendRefs:
+    - group: gateway.envoyproxy.io
+      kind: Backend
```

Comment:
this feature won't work with `Backend` because it doesnt have a `Service` with a clusterIP field, can you use one of the existing values e.g. 
```
      backendRefs:
        - name: infra-backend-v1
          port: 8080
``